### ISSUE TITLE
[FEATURE REQUEST] Attach files in the Android app

### ISSUE THREAD

**Issue by AlbinoGeek** [keybase/client#24333]
Currently, clicking the "Attach" button in the Android App allows you to `Upload Photo` or `Upload Video` but not arbitrary files. The desktop client allows you to upload arbitrary files, so why not the Android App?

**Comment by kf5grd** [comment]
You can upload any filetype in android by selecting it in your file explorer app and sharing it to the Keybase app. You'll have the option to attach it to a chat conversation, or upload it to a KBFS folder.

**Comment by AlbinoGeek** [comment]
Right, hence my request was specific to the `Attachment` button within a chat itself because a number of applications have an improperly implemented "Share to" or "Send" function, where I'd much rather use the native Android file browser (should it be available.)

**Comment by haukened** [comment]
But are other applications issues _this_ applications issue?

**Comment by AlbinoGeek** [comment]
Of course not, however, I don't see "third-party applications can do it" as a valid reason to turn down expanding on an otherwise partially implemented feature (attachments in chats). Currently, the attachments on the Android App specifically allow you to push a photo or video (and for some reason, as two different menu options) while leaving out other types of files entirely.

Yet, the desktop application allows you to attach arbitrary files to chats.

Arbitrary feature divergence doesn't make sense to me.

---

Now, if an argument were to be made about the permission model of the Android Operating System, this would be a perfectly valid reason -- but this has not been mentioned in this thread as of yet.
